### ISSUE TITLE
AppCenter Analtyics, IDFA, iOS 14.5 and AppTrackingTransparency

### ISSUE THREAD

**Issue by IainS1986** [microsoft/appcenter-sdk-apple#2295]
This isn't so much a Feature Request as a simple Question.

If we use AppCenter Analytics in an iOS app, does the IDFA for the user/device get tracked in anyway to AppCenter?

Do we need to ask permission from the user via AppTrackingTransparency if we use AppCenter?

https://developer.apple.com/documentation/apptrackingtransparency
Apples documentation states..

_"You must use the AppTrackingTransparency framework if your app collects data about end users and shares it with other companies for purposes of tracking across apps and web sites."_

**Comment by DmitriyKirakosyan** [comment]
Hi @IainS1986 

Thank you for reaching out to us!

The following docs describe the type of collected data and how it is used:
[Data Collected by App Center SDKs](https://docs.microsoft.com/appcenter/sdk/data-collected)
[AppStore Privacy Report](https://docs.microsoft.com/appcenter/gdpr/faq#app-store-privacy-report)
[General Data Protection Regulation](https://docs.microsoft.com/appcenter/gdpr/)

A similar report that can be useful to look at:
https://github.com/microsoft/appcenter-sdk-dotnet/issues/1482

To summarize, we do not collect any sensible user related information unless you send some using Analytics or Crashes SDKs. So regarding the `AppTrackingTransparency`, I would recommend to show the dialog with asking permissions only if you log any user personal data using the SDK.

Please feel free to reopen the issue is you have any additional questions or concerns.
